tools/hookgen/HookGen.cpp

```diff
@@ -214,8 +214,7 @@ HookGen::writeEventToPublicHeader(const char *name, const char *description, con
 	char *exampleBuf = (char *)malloc(bufSize + 1);
 	if (NULL != exampleBuf) {
 		/* If we fail to allocate room for the example just skip writing it */
-		int length = sprintf(exampleBuf, (char *)example, name, description, name, structName);
-		exampleBuf[length] = '\0';
```

Comment:
I removed this line because `snprintf()` always terminates the output with `'\0'`.